### DISCUSSION TITLE
Portal Engine - Downloading assets from the cart does not work and results into an error.

### DISCUSSION THREAD

**Discussion by jacobsyentl** [pimcore/pimcore#10858] [Q&A / Support]
Steps to reproduce:
- Open an asset portal
- Select multiple assets and bring them to shopping cart.
- Go to shopping cart and download the assets.
- Notification is shown that download is ready.
- Click on link to download.
Following error is shown:
![image](https://user-images.githubusercontent.com/9139261/142400064-73a97b39-8dd2-4dd3-b793-5f83fd9fb501.png)

**Comment by fashxp** [comment]
hmm, cannot  reproduce that on my end. on what version are you?

**Reply by jacobsyentl** [reply]
Pimcore: 10.2.4
Portal: 2.1.4

**Comment by githubsaf** [comment]
hi i have the same issue it was work correctly after i  change the index_prefix of elesticsearch i get this msg 'download resulted in zero files' by download all in shopping cart. 
can it be the cause ?

**Reply by fashxp** [reply]
did you rebuild the index after changing the index_prefix?

**Reply by githubsaf** [reply]
yes i did. i executed the both command line : 
 portal-engine:update:index 
portal-engine:update:process-index-queue
did i miss something ?

**Reply by fashxp** [reply]
did you also try `portal-engine:update:index-recreate` ?

**Reply by githubsaf** [reply]
yes i try it this also but i still get the same error

**Reply by fashxp** [reply]
can you try on latest version of portal engine? 
or are you already on latest version?

**Reply by fashxp** [reply]
also make sure, that all migrations are executed.
